lib/services/ntpd.pm

```diff
@@ -38,35 +48,40 @@ sub config_service {
 }
 
 sub enable_service {
-    systemctl('enable ntpd');
+    my ($service, $type) = get_service_name_type();
+    my $cmd = ($type eq 'SystemV') ? 'on' : 'enable';
```

Comment:
if you have func common_service_action, then this logic should put into func?

Reply:
changed.